cmake/thirdparty/BLTSetupHIP.cmake

```diff
@@ -87,14 +68,10 @@ else()
 endif()
 
 blt_import_library(NAME          blt_hip
-                   COMPILE_FLAGS ${_blt_hip_compile_flags}
```

Comment:
I just noticed this: _blt_hip_compile_flags is still set above but it is now unused. Is the fix for --rocm-path with crayftn no longer needed? @davidbeckingsale @white238

Reply:
@LLNL/blt my question still stands. We've noticed --rocm-path has disappeared from our compile lines.

Reply:
If something is being compiled as LANGUAGE HIP then why would you need the roam-path to be manually added (or for it to be guarded from CrayFTN)?

Reply:
I'm not sure, but something about the BLT update now causes some run time failures for us. We're experimenting to try to narrow it down. Adding `--rocm-path` back in doesn't fix it for us, but if it's really not needed, then all the code to set `_blt_hip_compile_flags` should be removed.

Reply:
Looks like it is not --rocm-path that was the issue. We had a dependency that was ignoring the BLT we were telling it to use and instead using its own internal BLT. We switched from using AMDGPU_TARGETS to CMAKE_HIP_ARCHITECTURES, but the old BLT was still relying on AMDGPU_TARGETS. So the code to set `_blt_hip_compile_flags` should probably be removed.